web/modules/custom/do_generated_content/src/Generator/TaxonomyTermGeneratorBase.php

```
@@ -0,0 +1,65 @@
+    foreach (static::NAMES as $name) {
+      if ($this->exists($name)) {
+        // A restored production database already carries most of these names,
+        // and a second term under the same name is indistinguishable to an
+        // editor. The existing term is deliberately left untracked: returning
+        // it here would hand real site taxonomy to the removal routine.
+        $this->helper::log('Reused existing %s term: %s', $this->getBundle(), $name);
+
+        continue;
+      }
+
+      $term = Term::create(['vid' => $this->getBundle(), 'name' => $name]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Drupal\taxonomy\Entity\Term' in method 'generate'. (undefined)

(StaticAccess)